pyproject.toml

```diff
@@ -3,7 +3,9 @@ build-backend = "scikit_build_core.build"
 requires = [
     "scikit-build-core",
     "pybind11",
-    "numpy>=2.0.0"
+    "numpy>=2.0.0",
+    "geopandas",
+    "shapely"
```

Comment:
This section is for packages necessary when we compile our C/C++ code. You probably don't need to add geopandas and shapely here.